sql/item_basic.sql

```diff
@@ -646,16 +645,16 @@ INSERT INTO `item_basic` VALUES (729,0,'bloodwood_log','bloodwood_log',12,0,43,0
 INSERT INTO `item_basic` VALUES (730,0,'piece_of_bloodwood_lumber','bldwd_lumber',12,0,43,0,1387);
 INSERT INTO `item_basic` VALUES (731,0,'aquilaria_log','aquilaria_log',12,0,43,0,0);
 INSERT INTO `item_basic` VALUES (732,0,'kapor_log','kapor_log',12,0,43,0,0);
-INSERT INTO `item_basic` VALUES (733,0,'piece_of_kapor_lumber','kapor_lumber',12,0,0,1,0);
-INSERT INTO `item_basic` VALUES (734,0,'chunk_of_thokcha_ore','thokcha_ore',12,0,0,1,0);
-INSERT INTO `item_basic` VALUES (735,0,'thokcha_ingot','thokcha_ingot',12,0,0,1,0);
```

Comment:
CREATE TABLE `item_basic` (
  `itemid` smallint(5) unsigned NOT NULL,
  `subid` smallint(4) unsigned NOT NULL DEFAULT '0',
  `name` tinytext NOT NULL,
  `sortname` tinytext NOT NULL,
  `stackSize` tinyint(2) unsigned NOT NULL DEFAULT '1',
  `flags` smallint(5) unsigned NOT NULL DEFAULT '0',
  `aH` tinyint(2) unsigned NOT NULL DEFAULT '99',
  `NoSale` tinyint(1) unsigned NOT NULL DEFAULT '0',
  `BaseSell` int(10) unsigned NOT NULL DEFAULT '0',